modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -94,7 +94,7 @@ AUI.add(
 
 					instance.set('timestamp');
 
-					if (event.src === SRC) {
+					if (event.src == SRC) {
```

Comment:
Intended? We [prefer triple equals anyway](https://github.com/liferay/eslint-config-liferay/issues/158).

Same further down in this file in various places.

Reply:
This was an automated revert and SF did not attempt a fix here... don't mind changing it... 🤷

Reply:
Glad to hear that SF didn't break it (we still need to turn on the lint, but every `==` we add to the codebase makes it harder to turn on the lint, which is why I always comment on them when I see them.)

Reply:
Yeah, I agree. This seems brittle enough that I don't want to touch anything else than necessary. I think we can revamp the whole thing after `7.3`. This is a huge source of issues and it's still AUI, so...

![just-sayin](https://user-images.githubusercontent.com/905006/89512587-b7350d00-d7d3-11ea-89df-0a7c4722b500.gif)